src/utils.ts

```diff
@@ -278,6 +278,7 @@ const standardErrors: {
   URIError,
   AggregateError,
   AbstractError,
+  ErrorRPCTimedOut: errors.ErrorRPCTimedOut,
```

Comment:
Doesn't `AbstractError` already cover that? What is this used for again?

Reply:
because the type of the error will be the name of the extended error (in this case, `ErrorRPCTimedout`) will not be `AbstractError`, so `toError` will not know to call the to error of the extended error class

Reply:
Can you show how it would be used?

Reply:
https://github.com/MatrixAI/js-rpc/blob/caa9bda08767f7d8e610f41a98656c8655a974be/src/utils.ts#L346